### PR TITLE
vagrant: fix NFS capability detection

### PR DESCRIPTION
**Pull request by renatoaguiar**

### CODE REVIEW

File: srcpkgs/vagrant/template [hunk 1]
```diff
@@ -11,6 +11,7 @@ license="MIT"
 homepage="https://www.vagrantup.com/"
 distfiles="https://github.com/mitchellh/${pkgname}/archive/v${version}.tar.gz"
 checksum=4f8ce6ede5d47a2be1ce46a29d22d41d1f4ba9d99fe9496b2424f98ae4dab2d6
+patch_args="-Np1"
```
Comment (maxice8) [inline srcpkgs/vagrant/template]: imo just adapt the patches

File: srcpkgs/vagrant/template [hunk 2]
```diff
@@ -11,6 +11,7 @@ license="MIT"
...
 nocross=yes
 
 archs="i686 x86_64"
```
Comment (maxice8) [inline srcpkgs/vagrant/template]: move it to  directly above hostmakedepends

Comment (renatoaguiar) [reply srcpkgs/vagrant/template]: done

Comment (renatoaguiar) [reply srcpkgs/vagrant/template]: @maxice8 PTAL

**Pull request merged**